BurnerB/Questioner-ADC-
# Bg fix travis 163135204

#### What does this PR do?
Fixes importation bug 
#### Description of Task to be completed?
fix importation in test_user endpoints to import from root instead of subfolders

#### How should this be manually tested?
Clone this repo onto yor machine using the link 
`https://github.com/BurnerB/Questioner-ADC-.git`
run `source .env` to set up virtual environment and set up flask
go to travis.io the build should now be working

#### What are the relevant pivotal tracker stories?
163135204
#### Screenshots (if appropriate)
#### Questions
![travis](https://user-images.githubusercontent.com/45785786/51016686-46d7f280-1582-11e9-8de6-d32d10dd7840.png)
